### DISCUSSION TITLE
Broadcast Range Control

### DISCUSSION THREAD

**Discussion by Theta6** [knijn/musicify#58] [Ideas]
Hi! I play on  a Stoneblock 3 Server (So each Person has their own Dimension) and Im wondering if there is a way to control what the Broadcast Range is for the Speaker and Music. As i would like to listen to Music as im going Dungeon Hunting or Roaming around my base with Friends.

If Not, Could you possibly make this happen?

**Comment by EmmaKnijn** [comment]
I could try and implement a volume thing, which wouldn't be *too* hard. One thing to consider is that Musicify is playing Music at it's loudest right now, so you can't up the range from what it is right now. I could also try to implement a syncplay thing if that is more what you want, which will broadcast the music over modem and be played on any receiving device, let me know what you would think is the best option.

**Reply by Theta6** [reply]
Yea, Syncplay would be Cool! If It's possible to have the option to Loop "Youcube" Tracks as well it would be Nice. Even if it Automatically Re-Executes The Musicify youcube <Link> Code Once the song has Finished!

A Bonus, Though i doubt Do-able, is making Spotify Playlists Playable Through CC.

**Reply by EmmaKnijn** [reply]
@Commandcracker Youcube is your part, could you implement a PR for the SyncPlay thing. I think just sending the audio data over modem would be the best
